Documentation/CoreArchitecture/Configuration/LocalConfiguration/Index.rst

```diff
@@ -152,3 +152,5 @@ MAIL
 
 SYS
   General options which may affect both the frontend and the backend.
+
+Details on the various configuration-options can be found in the install-tool as well as the TYPO3-source at typo3/sysext/core/Configuration/DefaultConfiguration.php. The documentation shown in the install-tool is automatically extracted from those comments of DefaultConfiguration.php.
```

Comment:
I would add a highlighting of the file:

```
:file:`typo3/sysext/core/Configuration/DefaultConfiguration.php`
```

That will highlight the file differently:
![image](https://cloud.githubusercontent.com/assets/354250/20061036/4b9314c2-a4fe-11e6-8ce9-eb7c29211dd7.png)